ingestion/supported-sources-and-formats.mdx

```diff
@@ -63,6 +63,15 @@ FORMAT [ DEBEZIUM | UPSERT | PLAIN ] ENCODE AVRO (
 
 Note that for `map.handling.mode = 'jsonb'`, the value types can only be: `null`, `boolean`, `int`, `string`, or `map`/`record`/`array` with these types.
 
+### Bytes
+
+RisingWave allows you to read data streams without decoding the data by using the `BYTES` row format. However, the table or source can have exactly one field of `BYTEA` data.
```

Comment:
Is it possible to have an example here? I don't get this exactly one field part. :)

Reply:
Try to sort these formats in alphabetical order, so move this one from down below to above 🤣 
<img width="537" alt="image" src="https://github.com/user-attachments/assets/dd4e7e5c-bbbe-4d1f-8aef-5e2e3d2befe2">